plutus-playground/plutus-playground-client/src/StaticData.purs

```diff
@@ -0,0 +1,166 @@
+module StaticData
+  ( editorContents
+  ) where
+
+editorContents :: String
+editorContents = """-- | Vesting scheme as a PLC contract
```

Comment:
I really think we should try and use some kind of file-embedding or data-files approach here rather than having this string literal which is going to be a pain forever.

Reply:
once we have a folder with an example in the correct place, courtesy of @j-mueller I will do this with the haskell side (to warm up the interpreter), not sure how to do it with purescript.

Reply:
We could use the template haskell fetched string in psgenerator to generate a file, @krisajenkins ?

Reply:
@j-mueller can you fix that tutorial, there is a code block incorrectly formatted and I just want to be sure its correct